deebot_client/map.py

```diff
@@ -55,6 +60,52 @@
     MapSetType.NO_MOP_ZONES: "#FFA500",
 }
 
+# SVG definitions referred by map elements
+_SVG_DEFS = svg.Defs(
+    elements=[
+        # Gradient used by Bot icon
+        svg.RadialGradient(
+            id="device_bg",
+            cx=svg.Length(50, "%"),
+            cy=svg.Length(50, "%"),
+            r=svg.Length(50, "%"),
+            fx=svg.Length(50, "%"),
+            fy=svg.Length(50, "%"),
+            elements=[
+                svg.Stop(offset=svg.Length(70, "%"), style="stop-color:#0000FF;"),
+                svg.Stop(offset=svg.Length(97, "%"), style="stop-color:#0000FF00;"),
+            ],
+        ),
+        # Bot circular icon
+        svg.G(
+            id=f"position_{PositionType.DEEBOT}",
+            elements=[
+                svg.Circle(r=5, fill="url(#device_bg)"),
+                svg.Circle(r=3.5, stroke="white", fill="blue", stroke_width=0.5),
+            ],
+        ),
+        # Charger pin icon (pre-flipped vertically)
+        svg.G(
+            id=f"position_{PositionType.CHARGER}",
+            transform=[svg.Scale(4, -4)],
+            elements=[
+                svg.Path(
+                    fill="#ffe605",
+                    d=[
+                        svg.M(1, -1.6),
+                        svg.C(1, -1.05, 0, 0, 0, 0),
+                        svg.c(0, 0, -1, -1.05, -1, -1.6),
+                        svg.c(0, -0.55, 0.45, -1, 1, -1),
+                        svg.c(0.55, 0, 1, 0.45, 1, 1),
+                        svg.Z(),
+                    ],
+                ),
+                svg.Circle(fill="white", r=0.7, cy=-1.6, cx=0),
+            ],
```

Comment:
```suggestion
            elements=[
                svg.Path(
                    fill="#ffe605",
                    d=[
                        svg.M(4, 6.4),
                        svg.C(4, 4.2, 0, 0, 0, 0),
                        svg.C(0, 0, -4, 4.2, -4, 6.4),
                        svg.C(-4, 8.6, -2.2, 10.4, 0, 10.4),
                        svg.C(2.2, 10.4, 4, 8.6, 4, 6.4),
                        svg.Z(),
                    ],
                ),
                svg.Circle(fill="white", r=2.8, cy=6.4, cx=0),
            ],
```

Applied tranformation